packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -220,7 +220,18 @@ const RealAccountSignup = ({
                     onConfirm={() => onErrorConfirm(local_props.state_value.error_code)}
                 />
             ),
-            title: () => localize('Add a real account'),
+            title: local_props => {
+                if (local_props.real_account_signup_target === 'add_crypto') {
+                    return localize('Create a cryptocurrency account');
+                } else if (local_props.real_account_signup_target === 'add_fiat') {
```

Comment:
why do we need this change?

Reply:
Make sure the Error Popup Component has the same title as the Modal Component